src/features/canvass/components/CanvassMap.tsx

```diff
@@ -15,23 +16,23 @@ import {
   AttributionControl,
   FeatureGroup,
   MapContainer,
+  Pane,
   Polygon,
   TileLayer,
 } from 'react-leaflet';
 
-import { ZetkinArea } from '../../areas/types';
+import CanvassMapOverlays from './CanvassMapOverlays';
 import { DivIconMarker } from 'features/events/components/LocationModal/DivIconMarker';
-import useCreateLocation from '../hooks/useCreateLocation';
-import useLocations from 'features/areaAssignments/hooks/useLocations';
 import getCrosshairPositionOnMap from '../utils/getCrosshairPositionOnMap';
-import getVisitState from '../utils/getVisitState';
-import MarkerIcon from '../utils/markerIcon';
-import { ZetkinAreaAssignment } from 'features/areaAssignments/types';
+import { getVisitPercentage } from '../utils/getVisitPercentage';
 import MapControls from 'features/areaAssignments/components/MapControls';
+import MarkerIcon from '../utils/markerIcon';
```

Comment:
Why is this module in `utils` and not in `components`, and why is it spelled `markerIcon` and not `MarkerIcon`?

Reply:
Good question, I'll do the changes! 👀